### DISCUSSION TITLE
How to compile *.zip?

### DISCUSSION THREAD

**Discussion by Fellow-baka** [tectonic-typesetting/tectonic#1099] [Q&A]
I'm failing to find a way to perform a standalone compilation of *.zip file (e.g. source downloaded from overleaf).
Is it possible?

The command I'm using is like this:
tectonic.exe -X compile temp_dep_new.zip

**Comment by rm-dr** [comment]
I don't think this is possible. Extract your zip to get to the TeX source inside.
